doc/rados/operations/add-or-rm-mons.rst

```diff
@@ -151,44 +145,45 @@ on ``mon.a``).
 Removing Monitors
 =================
 
-When you remove monitors from a cluster, consider that Ceph monitors use 
-Paxos to establish consensus about the master cluster map. You must have 
-a sufficient number of monitors to establish a quorum for consensus about 
-the cluster map.
+When monitors are removed from a cluster, it is important to remember
+that Ceph monitors use Paxos to establish consensus about the cluster
+map. Such consensus is possible only if the number of monitors is sufficient
+to establish quorum.
+
 
 .. _Removing a Monitor (Manual):
 
 Removing a Monitor (Manual)
 ---------------------------
 
-This procedure removes a ``ceph-mon`` daemon from your cluster.   If this
-procedure results in only two monitor daemons, you may add or remove another
-monitor until you have a number of ``ceph-mon`` daemons that can achieve a 
-quorum.
+The procedure in this section removes a ``ceph-mon`` daemon from the cluster. The
+procedure might result in a Ceph cluster that contains only two monitor daemons. To add more
```

Comment:
might result in insufficient monitors to maintain quorum, so plan carefully.  When swapping a new mon for old, add the new mon first, wait for quorum, then remove the old mon.  This ensures that quorum is not lost

Reply:
Accepted with minor alterations (none technical; all English-related).